cleo_sdk/CLEO_Utils.h

```diff
@@ -31,8 +31,7 @@ namespace CLEO
     OPCODE_READ_PARAM_INT()
     OPCODE_READ_PARAM_UINT()
     OPCODE_READ_PARAM_FLOAT()
-    OPCODE_READ_PARAM_STRING() // returns const char* to cstring. Uses internal buffer shared by all OPCODE_READ_PARAM_STRING calls
-    OPCODE_READ_PARAM_STRING_BUFF(_buffer, _bufferSize) // gets cstring into provided buffer, returns pointer to that buffer. Clamped to the buffer size, always null terminated
+    OPCODE_READ_PARAM_STRING_BUFF(_buffer, _bufferSize) // reads string from script, returns pointer to a null-terminated string or nullptr if error. Clamped to the buffer size, always null terminated
```

Comment:
No information about difference betweet he buffer and returned value. No info about the fact the buffer will not contain the text every time or the returned string ptr can be longer than provided buffer

Reply:
updated